content/en/docs/Installation/_index.md

```diff
@@ -338,14 +338,23 @@ kubectl logs -l app=kyverno -n <namespace>
 
 ### Flags
 
-The following flags are used to control the behavior of Kyverno and must be set in the Kyverno ConfigMap.
+The following flags are used to control the behavior of Kyverno and must be set in the Kyverno ConfigMap or Kyverno args.
 
 1. `excludeGroupRole`: excludeGroupRole role expected string with comma-separated group role. It will exclude all the group role from the user request. Default we are using `system:serviceaccounts:kube-system,system:nodes,system:kube-scheduler`.
 2. `excludeUsername`: excludeUsername expected string with comma-separated kubernetes username. In generate request if user enable `Synchronize` in generate policy then only kyverno can update/delete generated resource but admin can exclude specific username who have access of delete/update generated resource.
 3. `filterK8sResources`(deprecated): Kubernetes resources in the format "[kind,namespace,name]" where the policy is not evaluated by the admission webhook. For example --filterKind "[Deployment, kyverno, kyverno]" --filterKind "[Deployment, kyverno, kyverno],[Events, *, *]".
-4. `gen-workers`: the number of workers for processing generate policies concurrently. Default is set to 10.
-5. `background-scan`: the interval (like 30s, 15m, 12h) for background processing. Default is set to 1h.
+4. `genWorkers`: the number of workers for processing generate policies concurrently. Default is set to 10.
+5. `backgroundScan`: the interval (like 30s, 15m, 12h) for background processing. Default is set to 1h.
 6. `generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
+7. `kubeconfig`: specifies path to a kubeconfig. Only required if out-of-cluster.
+8. `serverIP`: specifies IP address where Kyverno controller runs. Only required if out-of-cluster.
+9. `profile`: setting this flag to 'true' will enable profiling.
+10. `profile-port`: specifies port to enable profiling at, defaults to 6060.
+11. `disableMetrics`: specifies whether (true/false) to enable exposing the metrics. Default is set to "false".
+12. `metrics-port`: specifies the port to expose prometheus metrics, default to port 8000.
```

Comment:
`metricsPort`?

Reply:
I have mentioned the issue with camelCase for port names in a previous comment, it is that Kubernetes does not allow camel case for port names.